repos/fdbt-site/src/pages/createCaps.tsx

```diff
@@ -1,34 +1,55 @@
 import React, { ReactElement, useState } from 'react';
-import CapTable from '../components/CapTable';
+import { ExpiryUnit } from '../interfaces/matchingJsonTypes';
 import CsrfForm from '../components/CsrfForm';
 import ErrorSummary from '../components/ErrorSummary';
-import FormElementWrapper from '../components/FormElementWrapper';
-import { CAPS_ATTRIBUTE } from '../constants/attributes';
-import { Cap, ErrorInfo, NextPageContextWithSession } from '../interfaces';
+import FormElementWrapper, { FormGroupWrapper } from '../components/FormElementWrapper';
+import { CREATE_CAPS_ATTRIBUTE } from '../constants/attributes';
+import { CapInfo, ErrorInfo, NextPageContextWithSession } from '../interfaces';
 import { isWithErrors } from '../interfaces/typeGuards';
 import { FullColumnLayout } from '../layout/Layout';
-import { getCsrfToken } from '../utils';
+import { getAndValidateNoc, getCsrfToken } from '../utils';
 import { getSessionAttribute } from '../utils/sessions';
+import { upperFirst } from 'lodash';
+import { getCapByNocAndId } from '../data/auroradb';
 
 const title = 'Create Caps - Create Fares Data Service';
 const description = 'Create caps page of the Create Fares Data Service';
 
+const daysOfWeek = ['monday', 'tuesday', 'wednesday', 'thursday', 'friday', 'saturday', 'sunday'];
+
 interface CreateCapsProps {
     errors: ErrorInfo[];
-    userInput: Cap[];
-    productName: string;
+    userInput?: CapInfo;
     csrfToken: string;
-    numberOfEntitesByDistancesToRender: number;
+    editId?: number;
 }
+export const isADayDuration = (duration: string | undefined, durationUnit: string | undefined): boolean =>
+    !!durationUnit && !!duration && !(durationUnit === 'hour' && Number(duration) <= 24);
+
+const CreateCaps = ({ errors = [], userInput, csrfToken, editId }: CreateCapsProps): ReactElement => {
+    let optionList: string[] = [];
+    Object.values(ExpiryUnit).map((unit) => {
+        optionList.push(unit);
+    });
+    optionList = optionList.filter((option) => option !== 'term');
```

Comment:
```suggestion
    const optionList: string[] = Object.values(ExpiryUnit).filter((option) => option !== 'term');
```

Reply:
On line 31 we have a map - maps should only be returning values, to build up a new array. If we're just pushing to an array, then we can just return the value.